Gemfile

```diff
@@ -3,6 +3,9 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in hydra-works.gemspec
 gemspec
 
+gem 'active-fedora', github:'projecthydra/active_fedora', branch: 'master'
```

Comment:
@jcoyne definitely — I'm trying to get memberOf functionality in CurationConcerns today for discussion.  We'll want to have released versions of hydra-pcdm, hydra-works, and hydra-derivatives.